api.py

```diff
@@ -71,14 +122,131 @@ def get(self):
 		self.response.headers['Content-Type'] = 'application/json'
 		self.response.write(json.dumps(output))
 
+class RankAPI(webapp2.RequestHandler):
+	def get(self, rank_type, id):
```

Comment:
Why parts of the path rather than parameters?

Reply:
See my other comment on URL path structure, I think I will remove the rank_type parameter though, since per your other comment that is probably not the best way to handle that interface.

Reply:
See my reply above about URL path structure.